api/routes_database.go

```diff
@@ -61,5 +64,8 @@ func (api *API) getUploadsForAddress(c *gin.Context) {
 		"user":    user,
 	}).Info("specific uploads from database requested")
 
-	c.JSON(http.StatusFound, gin.H{"uploads": uploads})
+	c.JSON(http.StatusOK, gin.H{
```

Comment:
just a thought, but might be cleaner to have a utility function:

```go
package utils
func Respond(c *gin.Context, status int, body gin.H) {
    body["response"] = status
    c.JSON(status, body)
}
```

then:

```go
utils.Respond(c, http.StatusOK, gin.H{"response": uploads})
```

less repetition, and no need to worry about mismatching header/body